consensus/hotstuff/timeoutcollector/aggregation_test.go

```diff
@@ -171,15 +171,15 @@ func TestTimeoutSignatureAggregator_VerifyAndAdd(t *testing.T) {
 			// test thread safety
 			go func(i int, sig crypto.Signature) {
 				defer wg.Done()
-				weight, err := aggregator.VerifyAndAdd(ids[i].NodeID, sigs[i], highQCViews[i]) // same signature for same index
+				weight, err := aggregator.VerifyAndAdd(ids[i].NodeID, sigs[i], newestQCViews[i]) // same signature for same index
 				// weight should not change
 				require.Equal(t, expectedWeight, weight)
 				require.True(t, model.IsDuplicatedSignerError(err))
-				weight, err = aggregator.VerifyAndAdd(ids[i].NodeID, sigs[(i+1)%signersNum], highQCViews[(i+1)%signersNum]) // different signature for same index
+				weight, err = aggregator.VerifyAndAdd(ids[i].NodeID, sigs[(i+1)%signersNum], newestQCViews[(i+1)%signersNum]) // different signature for same index
 				// weight should not change
 				require.Equal(t, expectedWeight, weight)
 				require.True(t, model.IsDuplicatedSignerError(err))
-				weight, err = aggregator.VerifyAndAdd(ids[(i+1)%signersNum].NodeID, sigs[(i+1)%signersNum], highQCViews[(i+1)%signersNum]) // different signature for same index
+				weight, err = aggregator.VerifyAndAdd(ids[(i+1)%signersNum].NodeID, sigs[(i+1)%signersNum], newestQCViews[(i+1)%signersNum]) // different signature for same index
 				// weight should not change
 				require.Equal(t, expectedWeight, weight)
 				require.ErrorAs(t, err, &model.ErrInvalidSignature)
```

Comment:
## ⚠️ 

based on Jordan's earlier comment, I think it would be better to write here
```suggestion
				require.ErrorIs(t, err, model.ErrInvalidSignature)
```